src/function.rs

```diff
@@ -1,7 +1,7 @@
 use crate::common::*;
 
+use camino::Utf8Path;
```

Comment:
You can add this import to `common.rs`

Reply:
I'm not sure if I added it in the correct way